scenarios/19c_The_Oracle.cfg

```diff
@@ -576,7 +576,7 @@
                     id=Matthias Ramon
                 [/criteria]
                 value={ON_DIFFICULTY 7 8 9}
-            [/goal]
+            [/goal] #
```

Comment:
What is the hash for?

Reply:
that was just to get the diffs to render properly on GitHub; I can remove them again if they're distracting